Resources/Locale/en-US/datasets/names/arachnid_first.ftl

```diff
@@ -0,0 +1,74 @@
+names-arachnid-first-dataset-1 = Amblyocarenum
+names-arachnid-first-dataset-2 = Aname
+names-arachnid-first-dataset-3 = Aname
```

Comment:
Any idea why Aname is duplicated twice (it's also duplicated in master)?

Reply:
No idea, it just was there. Should be fixed later I think